src/renderer/dx/DxRenderer.cpp

```diff
@@ -2115,3 +2117,18 @@ void DxEngine::SetSelectionBackground(const COLORREF color) noexcept
                                         GetBValue(color) / 255.0f,
                                         0.5f);
 }
+
+// Routine Description:
+// - Changes the antialiasing mode of the renderer. This must be called before
+//   _PrepareRenderTarget, otherwise the renderer will default to
+//   D2D1_TEXT_ANTIALIAS_MODE_GRAYSCALE. This has only been tested with

```

Comment:
it is no longer true that it has been tested with only those :smile: right?

Reply:
oof okay fine